content/docs/04-clusters/01-public-cloud/01-aws.md

```diff
@@ -946,6 +963,33 @@ To create an AWS cloud account using STS credentials follow the steps below:
 </Tabs.TabPane>
 </Tabs>
 
+## Create AWS US Gov Cloud Account
+
+ To deploy the AWS US Gov Clusters in the Palette Platform, having the AWS US Gov Cloud Account registered to the Palette Platform is a significant prerequisite. This section guides you on creating an AWS US Gov cloud account in the Palette console.
+
+To create an AWS US Gov cloud account using access credentials follow the steps below:
+
+1. In the AWS console, create the [four policies listed](/clusters/public-cloud/aws#awscloudaccountpermissions). Assign this role to the root user or the IAM user to be used from Palette.
+
+
+2. Login to Palette console as Tenant Admin.
+
+
+3. Go to `Tenant Settings` > `Cloud Accounts` and click `+Add AWS Account`.
+
+
+4. In the cloud account creation wizard give the following information:
+   * **Account Name:** Custom name for the Cloud Account.
+   * **Description:** Optional description for the cloud account.
+   * **Partition:** Choose **AWS Gov Cloud** from the drop down-menu.
+   * **Credentials:**
+       * AWS Access key
+       * AWS Secret access key
+
+
+5. Click **Validate** button to validate the credentials, once the credentials are validated, a toggle button appears as `Add IAM Policies`, once the option is toggled, a `dropdown appears` listing all the policies attached in the AWS console. User can make the choice of policies from the dropdown and attach it to the cloud account being created.
```

Comment:
```suggestion
5. Click on the **Validate** button to validate the credentials.  Once the credentials are validated, a toggle button appears as **Add IAM Policies**. 

 6. Toggle the button to add IAM policies. Add the required IAM policies you previously created. 
```
Step five is packed with multiple actions, break out each step into its own bullet.